resources/public/api/conf/1.0/schemas/businesses/create_request.json

```diff
@@ -104,36 +104,37 @@
       "example": "2020-03-24"
     },
     "businessAddressLineOne": {
-      "description": "First line of the business address.",
+      "description": "First line of the business address.\n\nForbidden for property businesses.\n\nMandatory for self-employment businesses.",
       "type": "string",
       "maximum": 35,
       "example": "1 ABC Rd."
     },
     "businessAddressLineTwo": {
-      "description": "Second line of the business address.",
+      "description": "Second line of the business address.\n\nForbidden for property businesses.",
       "type": "string",
       "maximum": 35,
       "example": "London"
     },
     "businessAddressLineThree": {
-      "description": "Third line of the business address.",
+      "description": "Third line of the business address.\n\nForbidden for property businesses.",
       "type": "string",
       "maximum": 35,
       "example": "Greater London"
     },
     "businessAddressLineFour": {
-      "description": "Fourth line of the business address.",
+      "description": "Fourth line of the business address.\n\nForbidden for property businesses.",
       "type": "string",
       "maximum": 35,
       "example": "United Kingdom"
     },
     "businessAddressPostcode": {
-      "description": "Business postcode.",
+      "description": "Business postcode. Forbidden for property businesses.\n\nMandatory for self-employment businesses if businessAddressCountryCode \"GB\" is provided.",
```

Comment:
Maybe we should put a new line between the description of the field and where it is forbidden/allowed, like we have for the other ones, i.e. change `Business postcode. Forbidden for property businesses.` to `Business postcode.\n\nForbidden for property businesses.\n\n...`